src/helper.py

```diff
@@ -14,9 +18,33 @@
 from sklearn.neural_network import MLPClassifier
 
 try:
-    from config import MOCK_COURSE_INFO_CSV, ICON_ORDER_PATH
+    from config import (
+        MOCK_COURSE_INFO_CSV,
+        ICON_ORDER_PATH,
+        COURSE_WORK_EXTRACTED_INFO,
+    )
 except ImportError:
-    from .config import MOCK_COURSE_INFO_CSV
+    from .config import (
+        MOCK_COURSE_INFO_CSV,
+        ICON_ORDER_PATH,
+        COURSE_WORK_EXTRACTED_INFO,
+    )
+
+# Initialize OpenAI API with your API key
+openai.api_key = "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"
```

Comment:
Please set the key as an environmental variable, and give instructions on how to set it in README.md